### PR TITLE
Create README.md for mqtt output

### PR DESCRIPTION
**Pull request by denzuko**
### Required for all PRs:

- [  ] Signed [CLA](https://influxdata.com/community/cla/).
- [  ] Associated README.md updated.
- [  ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/outputs/mqtt/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# MQTT Producer Output Plugin
+
+This plugin writes to a [MQTT Broker](http://http://mqtt.org/) acting as a mqtt Producer.
+
+```
+[[outputs.mqtt]]
+  ## URLs of mqtt brokers
+  brokers = ["localhost:1883"]
```
Comment (danielnelson) [inline plugins/outputs/mqtt/README.md]: Should be `servers`

File: plugins/outputs/mqtt/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* `username`: The username to connect MQTT server.
+* `password`: The password to connect MQTT server.
+* `client_id`: The unique client id to connect MQTT server. If this paramater is not set then a random ID is generated.
+* `timeout`: Timeout for write operations. default: 5s
+* `ssl_ca`: SSL CA
+* `ssl_cert`: SSL CERT
+* `ssl_key`: SSL key
+* `insecure_skip_verify`: Use SSL but skip chain & host verification (default: false)
+* `data_format`: [About Telegraf data formats](https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md)
+* `topic_suffix`: Which, if any, method of calculating `kafka` topic suffix to use.
```
Comment (danielnelson) [inline plugins/outputs/mqtt/README.md]: This option doesn't exist

**Comment by denzuko** [comment]
@danielnelson Corrections made.

**Comment by danielnelson** [comment]
Thank you!

**Pull request merged**